src/Config/SettingsTab.ts

```diff
@@ -233,6 +233,24 @@ export class SettingsTab extends PluginSettingTab {
                 });
             });
 
+        new Setting(containerEl)
+            .setName('Additional filename date format as Scheduled date for undated tasks')
+            .setDesc(
+                SettingsTab.createFragmentWithHTML(
+                    'An additional date format that Tasks plugin will recogize when using the file name as the Scheduled date for undated tasks.',
+                ),
+            )
+            .addText((text) => {
+                const settings = getSettings();
+
+                text.setPlaceholder('YYYY-MM-DD')
```

Comment:
I wasn't quite sure what to make the placeholder - In other plugins (like 'Periodic Notes') the placeholder shows the default date format. So I intended to do the same here. But I could also update it to show an "example" date format.

Reply:
That is a good point. I think it would still help to show a different format though, for clarity.

Reply:
I tried doing this, but it's difficult to update dynamically. I tried refreshing display() whenever the setting was updated, which worked, but it would remove the cursor from the text box and made the experience bad. Not sure what more could work without refactoring setDesc().

Reply:
Ok. Thanks for trying.